docs/semgrep-assistant/getting-started.md

```diff
@@ -180,10 +180,45 @@ By switching from Semgrep's key to your key, note that you lose access to the fo
 - Semgrep's [Zero Data Retention agreement](/semgrep-assistant/privacy) that prevents OpenAI from saving input or output data.
 - Semgrep paying for the cost of your AI usage.
 
-### Additional AI providers
+## Use your AI provider
 
 If you would like access to the following AI providers for use with your Semgrep organization, click the <i class="fa-regular fa-envelope"></i> **icon** next to the AI provider of your choice to request access:
 
 - Azure OpenAI
 - AWS Bedrock
 - Google Gemini
+
+### Azure OpenAI
+
+To use Azure OpenAI with Semgrep Assistant, you must retrieve the endpoint URL and API key for your model from Azure, then provide it to Semgrep.
+
+1. To retrieve the endpoint URL and API key from Azure:
+   1. Log in to Azure OpenAI Studio.
+   2. Navigate to **Deployments**, and select the deployment you want to use.
+   3. In **Endpoint**, find and copy both the **Target URI** and the **API key**. You will provide both values to Semgrep.
+2. To configure Semgrep to use Azure OpenAI:
+   1. Sign in to [Semgrep AppSec Platform](https://semgrep.dev/login?return_path=/manage/projects) and navigate to [<i class="fa-solid fa-gear"></i> **Settings > Deployment**](https://semgrep.dev/orgs/-/settings).
+   2. In the **Assistant** section, click the <i class="fa-solid fa-gear"></i> **icon** next to **AI provider**.
+   3. Select **Azure OpenAI**.
+   4. Paste the **Target URI** you copied from Azure into **Your Azure OpenAI Endpoint**.
+   5. Paste the API key you copied from Azure into **Your Azure OpenAI API key**.
+   6. Click **Save** to proceed.
+
+Your Azure OpenAI model is now configured for use in Semgrep.
+
+:::note
+You can switch to a different Azure OpenAI model anytime by repeating these configuration steps using the Target URI and API key for the new model.
+:::
+
+#### Troubleshoot issues with Azure OpenAI
+
+If you see **Error 429 - Max Tokens Exceeded**:
+
+1. Go to **Azure OpenAI Studio > Deployments** and select your active deployment.
+1. Under **Details**, click **Edit** and increase the **Tokens per Minute Rate Limit** to the maximum value.
```

Comment:
```suggestion
1. Under **Details**, click **Edit** and increase the **Tokens per Minute Rate Limit** to the maximum value.
```

Do we know what the maximum value is? I'm guessing this is a slider, from how it sounds?

Reply:
(It also sounds like the max val is dependent on the plan.)

Reply:
I could be [misreading this doc completely](https://learn.microsoft.com/en-us/azure/ai-services/openai/quotas-limits), but it looks highly variable